src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -257,7 +264,14 @@ def __call__(
             uncond_embeddings = self.text_encoder(uncond_input.input_ids.to(self.device))[0]
 
             # duplicate unconditional embeddings for each generation per prompt
-            uncond_embeddings = uncond_embeddings.repeat_interleave(batch_size * num_images_per_prompt, dim=0)
+            if self.device.type == "mps":
```

Comment:
can we always use this, instead of checking for device here ?

Reply:
+1 - I think this makes more sense